dozer-types/src/errors/types.rs

```diff
@@ -59,4 +59,12 @@ pub enum DeserializationError {
     Utf8(#[from] std::str::Utf8Error),
     #[error("Failed to convert type due to json numbers being out of the f64 range")]
     F64TypeConversionError,
+    #[error("Unknown SSL mode: {0}")]
+    UnknownSslMode(String),
+    #[error("Invalid connection url for Postgres configuration")]
+    InvalidConnectionUrl,
```

Comment:
We need to include the error source and tell the user why the connection url is invalid

Reply:
sure